src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/contexts/ResourceContext.java

```diff
@@ -53,33 +40,23 @@ public static ResourceContext generic() {
   public ResourceContextMessage buildContextMessage() {
     final ResourceContextMessage.Builder contextMessageBuilder = ResourceContextMessage
         .newBuilder();
-    contextMessageBuilder.setState(state.ordinal());
+    contextMessageBuilder.setState(getState().ordinal());
     return contextMessageBuilder.build();
   }
 
   public static ResourceContext buildResourceContextFromMessage(ResourceContextMessage message) {
     return new ResourceContext(Resources.State.values()[message.getState()]);
   }
 
-  @Override
-  public String toString() {
-    return this.state.toString();
-  }
-
   public List<Field<?>> getSqlSchema() {
     List<Field<?>> schemaList = new ArrayList<>();
-    schemaList.add(DSL.field(DSL.name(SQL_SCHEMA_CONSTANTS.STATE_COL_NAME), String.class));
+    schemaList.add(DSL.field(DSL.name(STATE_COL_NAME), String.class));
     return schemaList;
   }
 
   public List<Object> getSqlValue() {
     List<Object> valueList = new ArrayList<>();
-    valueList.add(this.state.toString());
+    valueList.add(getState().toString());
     return valueList;
   }
-
-  public static class SQL_SCHEMA_CONSTANTS {
```

Comment:
let's not remove this subclass. This CONSTANTS class defines the SQL schema and we might have multiple columns in the future

Reply:
Restored subclass